data/mods/Aftershock/mutations/mutations.json

```
@@ -9,6 +9,67 @@
     "player_display": false,
     "threshold": true
   },
+  {
+    "type": "mutation",
+    "id": "PROF_POLICE",
+    "name": { "str": "Local System Security" },
+    "points": 0,
+    "description": "You are a duly sworn law enforcement officer, with jurisdiction that technically extends to the Salus System.  Whether you can find anyone who that matters to is questionable.",
+    "valid": false,
+    "purifiable": false,
+    "profession": true
+  },
+  {
+    "type": "mutation",
+    "id": "PROF_SWAT",
+    "name": { "str": "Pinkerton" },
+    "points": 0,
+    "description": "You are a corporate security professional, with jurisdiction that technically extends to the Salus System.  Whether you can find anyone who that matters to is questionable.",
```

```suggestion
    "description": "You are an UICA affiliated private-security professional, with jurisdiction that technically extends to the Salus System.  Whether you can find anyone who that matters to is questionable.",
```

Looking at this traits hardcoded efects it might make more sense to keep them state linked.